src/pages/index.js

```diff
@@ -0,0 +1,259 @@
+import "./index.css";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+import { handleSubmit } from "../utils/helpers";
+import Api from "../utils/Api.js";
+
+let selectedCard, selectedCardId;
+
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9761a594-3fe1-46bc-8b2b-b1c551adb983",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, users]) => {
+    cards.reverse();
+    cards.forEach((card) => {
+      renderCard(card);
+    });
+
+    profileImage.src = users.avatar;
+    profileName.textContent = users.name;
+    profileDescription.textContent = users.about;
+  })
+  .catch(console.error);
+
+// Universal elements
+const closeButtons = document.querySelectorAll(".modal__close-btn");
+
+// Profile elements
+const editProfileButton = document.querySelector(".profile__edit-btn");
+const profileImage = document.querySelector(".profile__avatar");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const cardModalButton = document.querySelector(".profile__add-btn");
+const avatarModalButton = document.querySelector(".profile__avatar-btn");
+
+// Card form elements
+const editProfileModal = document.querySelector("#edit-modal");
+const editProfileForm = editProfileModal.querySelector("#edit-profile-form");
+const nameInput = editProfileModal.querySelector("#profile-name-input");
+const descriptionInput = editProfileModal.querySelector(
+  "#profile-description-input"
+);
+const cardModal = document.querySelector("#card-modal");
+const cardModalForm = cardModal.querySelector("#card-form");
+const cardSubmitButton = cardModal.querySelector(".modal__submit-btn");
+const imageInput = cardModal.querySelector("#card-link-input");
+const titleInput = cardModal.querySelector("#card-caption-input");
+
+// Avatar form elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarModalForm = avatarModal.querySelector("#edit-avatar-form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms